packages/e2e/src/specs/overview/mc_version_check.e2e.ts

```diff
@@ -0,0 +1,40 @@
+import NavigationTabs from "../../page-objects/tabs.page"
+import settingsPage from "../../page-objects/settings.page"
```

Comment:
Maybe `settingsPage` should be changed to `SettingsPage` (the first letter is a capital letter) to follow the convention?